src/python/pants/backend/codegen/tasks/simple_codegen_task.py

```diff
@@ -95,7 +95,7 @@ def execute(self):
     with self.invalidated(targets,
                           invalidate_dependents=True,
                           fingerprint_strategy=self.get_fingerprint_strategy()) as invalidation_check:
-      for vts in invalidation_check.invalid_vts:
+      for vts in invalidation_check.invalid_vts_partitioned:
```

Comment:
Interesting.  Maybe this is the same as this issue?  https://github.com/pantsbuild/pants/issues/1495